subsys/net/ip/Kconfig

```diff
@@ -412,6 +412,18 @@ config NET_TCP_RETRY_COUNT
 	  Should a retransmission timeout occur, the receive callback is
 	  called with -ECONNRESET error code and the context is dereferenced.
 
+config NET_TCP_RANDOMIZED_RTO
+	bool "Use and randomized retransmission time"
```

Comment:
Is there a typo here, in the sentence `Use and randomized....`, the "and" sounds wrong here.

Reply:
Thanks, should be "a" instead of "and", fixed

Reply:
It doesn't seem to be fixed thou, I still see "Use and randomized..."